Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks_bottles.yml

```diff
@@ -467,36 +467,3 @@
   - type: Drink
   - type: Sprite
     sprite: Objects/Consumable/Drinks/cream.rsi
-
-- type: entity
-  parent: DrinkBottleBaseFull
-  id: DrinkMilkCarton
-  name: milk
-  description: An opaque white liquid produced by the mammary glands of mammals.
-  components:
-  - type: SolutionContainerManager
-    solutions:
-      drink:
-        maxVol: 100
-        reagents:
-        - ReagentId: Milk
-          Quantity: 100
-  - type: Sprite
-    sprite: Objects/Consumable/Drinks/milk.rsi
-
-- type: entity
-  parent: DrinkBottleBaseFull
-  id: DrinkSoyMilkCarton
-  name: soy milk
-  description: White and nutritious soy goodness!
-  components:
-  - type: SolutionContainerManager
-    solutions:
-      drink:
-        maxVol: 100
-        reagents:
-        - ReagentId: MilkSoy
-          Quantity: 100
-  - type: Drink
-  - type: Sprite
-    sprite: Objects/Consumable/Drinks/soymilk.rsi
```

Comment:
Why remove this entirely? The commit says duplicate milk bag, but I can't find the alleged other one.

Reply:
These entities are basically duplicates of these two entities (which are used everywhere):
https://github.com/space-wizards/space-station-14/blob/b474b391c0736f885e3a69fd193de6755478cd98/Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml#L209-L239
But now I intend to revert this removal, as it causes map-related test error and I don't want to break something.